packages/devtools/lib/src/timeline/frames_bar_chart.dart

```diff
@@ -52,12 +52,14 @@ class PlotlyDivGraph extends CoreElement {
   PlotlyDivGraph(this.framesBarChart, this.frame, [bool datum = true])
       : _processDatum = datum,
         super('div') {
-    element.id = 'graph';
+    element.id = FRAME_GRAPH;
     element.style
       ..height = '100%'
       ..width = '100%';
   }
 
+  static const String FRAME_GRAPH = 'graph_frame_timeline';
```

Comment:
nit: rename constant to
`frameGraphId`
At some point the dart style guide changed to making constants lower camel case.

Reply:
Done

Reply:
Done